src/main/java/com/umc/withme/controller/MeetController.java

```diff
@@ -41,4 +43,23 @@ public ResponseEntity<DataResponse<MeetCreateResponse>> createMeet(
                 HttpStatus.OK
         );
     }
+
+    /**
+     * 모임 단건 조회 API
+     * 모임 id로 모임을 1건 조회한다.
+     *
+     * @param meetId
+     * @return 조회한 모임의 정보를 담은 MeetInfoGetResponse를 data에 담아서 반환한다.
+     */
+    @GetMapping("/meet")
+    public ResponseEntity<DataResponse<MeetInfoGetResponse>> getMeet(@RequestParam Long meetId){
```

Comment:
```suggestion
    @GetMapping("/meet/{meetId}")
    public ResponseEntity<DataResponse<MeetInfoGetResponse>> getMeet(@PathVariable Long meetId){
```
이 API는 query string보다는 path variable이 더 낫지 않을까요?

Reply:
생각해보니 그게 더 적합할 것 같네요. 해당 사항 반영했습니다!